openghg/store/_flux_timeseries.py

```diff
@@ -0,0 +1,273 @@
+            species: Species name
+            domain: Region for Flux timeseries
+            source: Source of the emissions data, e.g. "energy", "anthro", default is 'anthro'.
+            region: Region/Country of the CRF data
+            domain: Geographic domain, default is 'None'. Instead region is used to identify area
+            database: Name of database source for this input (if relevant)
+            database_version: Name of database version (if relevant)
+            model: Model name (if relevant)
+            source_format : Type of data being input e.g. openghg (internal format)
+            period: Period of measurements. Only needed if this can not be inferred from the time coords
+            If specified, should be one of:
+                - "yearly", "monthly"
+                - suitable pandas Offset Alias
+                - tuple of (value, unit) as would be passed to pandas.Timedelta function
+            continuous: Whether time stamps have to be continuous.
+            if_exists: What to do if existing data is present.
+                - "auto" - checks new and current data for timeseries overlap
+                   - adds data if no overlap
+                   - raises DataOverlapError if there is an overlap
+                - "new" - just include new data and ignore previous
+                - "combine" - replace and insert new data into current timeseries
+            save_current: Whether to save data in current form and create a new version.
+                - "auto" - this will depend on if_exists input ("auto" -> False), (other -> True)
+                - "y" / "yes" - Save current data exactly as it exists as a separate (previous) version
+                - "n" / "no" - Allow current data to updated / deleted
+            overwrite: Deprecated. This will use options for if_exists="new".
+            force: Force adding of data even if this is identical to data stored.
+            compressor: A custom compressor to use. If None, this will default to
+                `Blosc(cname="zstd", clevel=5, shuffle=Blosc.SHUFFLE)`.
+                See https://zarr.readthedocs.io/en/stable/api/codecs.html for more information on compressors.
+            filters: Filters to apply to the data on storage, this defaults to no filtering. See
+                https://zarr.readthedocs.io/en/stable/tutorial.html#filters for more information on picking filters.
+            optional_metadata: Allows to pass in additional tags to distinguish added data. e.g {"project":"paris", "baseline":"Intem"}
+        Returns:
+            dict: Dictionary of datasource UUIDs data assigned to
+        """
+        from openghg.types import FluxTimeseriesTypes
+
+        from openghg.util import (
+            clean_string,
+            load_flux_timeseries_parser,
+            check_if_need_new_version,
+        )
+
+        species = clean_string(species)
+        source = clean_string(source)
+        if domain:
```

Comment:
The `region` keyword should also be passed to the `clean_string` function for consistency.